modules/appeals_api/app/services/appeals_api/events/status_updated.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module Events
+    class StatusUpdated
+      def initialize(opts)
```

Comment:
i see some of these options are required, any particular reason to not use keyword arguments here?

Reply:
StatusUpdated (or any other callback class) would never actually be called directly from the application - it's proxied from Events::Handler. As a result, the interface requires opts, but the error would be functionally equivalent in a scenario where it's not included - instead of 'missing keyword opts' (which, as I write this, sounds better), it'd be `given 1 argument, expected 0`. I'll add it.

Reply:
Actually, in the interface where the method is called, it is added as a kwarg.

`AppealsApi::Events::Handler.new(opts: opts, event_type: :event_type).handle!` requires that opts is included at the public interface level.